plugin/discovery/get.go

```diff
@@ -28,7 +29,7 @@ import (
 
 const protocolVersionHeader = "x-terraform-protocol-version"
 
-var releaseHost = "https://releases.hashicorp.com"
+var ReleaseHost = "https://releases.hashicorp.com"
```

Comment:
Exposing this as a variable makes me nervous that a future maintainer will be tempted to mutate it to customize the repository URL, rather than introducing a "real" API for that.

Could we instead make this a function that returns the private variable, thus making it clear that this is a read-only interface?